code/game/objects/items/weapons/clown_items.dm

```diff
@@ -66,6 +66,10 @@
 	else if(ishuman(target) && user.zone_sel && user.zone_sel.selecting == "mouth")
 		user.visible_message("<span class='warning'>\the [user] washes \the [target]'s mouth out with [src.name]!</span>", "<span class='notice'>You wash \the [target]'s mouth out with [src.name]!</span>") //washes mouth out with soap sounds better than 'the soap' here
 		return
+	else if(istype(target, /obj/structure/window))
+		user << "<span class='notice'>You clean \the [target.name].</span>"
+		target.color = initial(target.color)
+		target.SetOpacity(0)

```

Comment:
They are, mostly for the chapel confession rooms, but some mapper probably used them elsewhere. Anyhow, that's not an excuse to not add a check for them.